index.js

```diff
@@ -164,31 +180,76 @@ DB.prototype.del = function (key, opts, cb) {
   }
   if (!opts) opts = {}
   cb = once(cb || noop)
-  self._del(key, opts, function (err, rows) {
-    if (err) return cb(err)
-    self.batch(rows, opts, function (err, nodes) {
-      if (err) cb(err)
-      else cb(null, nodes[0])
-    })
+
+  var rows = [
+    {
+      type: 'del',
+      key: key,
+      links: opts.links || undefined
```

Comment:
I'm not sure why I had this, to be honest. Tests still pass without it. The only thing my code would do is ensure that non-truthy values like `null` or `0` would be mapped to `undefined`. I'm good with adding a sanity check though to ensure that `opts.links` is strictly either `undefined` or an array.